apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -0,0 +1,216 @@
+                    if (operation === 'create') {
+                        const value = resolvedData[fieldPath]
+                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
+
+                        if (lastMeterReading && lastMeterReading.value2 > value)
+                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
+                    }
+                },
+            },
+        },
+
+        value3: {
+            schemaDoc: 'Meter reading value of tariff 3',
+            type: Integer,
+            hooks: {
+                validateInput: async ({ context, operation, resolvedData, fieldPath, addFieldValidationError }) => {
+                    if (operation === 'create') {
+                        const value = resolvedData[fieldPath]
+                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
+
+                        if (lastMeterReading && lastMeterReading.value3 > value)
+                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
+                    }
+                },
+            },
+        },
+
+        value4: {
+            schemaDoc: 'Meter reading value of tariff 4',
+            type: Integer,
+            hooks: {
+                validateInput: async ({ context, operation, resolvedData, fieldPath, addFieldValidationError }) => {
+                    if (operation === 'create') {
+                        const value = resolvedData[fieldPath]
+                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
+
+                        if (lastMeterReading && lastMeterReading.value4 > value)
+                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
+                    }
+                },
+            },
+        },
+
+        sectionName: {
+            schemaDoc: 'Section name/number of an apartment building (property). You need to take from Property.map',
+            type: Text,
+        },
+
+        floorName: {
+            schemaDoc: 'Floor of an apartment building (property). You need to take from Property.map',
+            type: Text,
+        },
+
+        unitName: {
+            schemaDoc: 'Flat number / door number of an apartment building (property). You need to take from Property.map',
+            type: Text,
+            isRequired: true,
+        },
+
+        client: {
+            schemaDoc: 'Inhabitant/customer/person who has a problem or want to improve/order something. Not null if we have a registered client',
+            type: Relationship,
+            ref: 'User',
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
+        },
+
+        contact: {
```

Comment:
It's better to use it from `fields` like sender

Reply:
done in 719225f